Content/0.3.3 - Account History/accountHistory.json

```diff
@@ -0,0 +1,34 @@
+{
+	"accountHistory_title": "History",
+    "accountHistory_transactionClassGeneral": "General",
+    "accountHistory_transactionClassTransfer": "Transfer",
+    "accountHistory_transactionClassStaking": "Staking",
+    "accountHistory_transactionClassUnstaking": "Unstaking",
+    "accountHistory_transactionClassClaim": "Claim",
+    "accountHistory_transactionClassContribute": "Contribute",
+    "accountHistory_transactionClassRedeem": "Redeem",
+    "accountHistory_transactionClassAccountSettings": "Deposit Settings",
+    "accountHistory_transactionClassOther": "Other",
+    "accountHistory_filters_title": "Filter",
+    "accountHistory_filters_clearAll": "Clear All",
+    "accountHistory_filters_showResultsButton": "Show Results",
+    "accountHistory_filters_assetTypeLabel": "Type of Asset",
+    "accountHistory_filters_assetTypeTokenShowAll": "+ Show All Tokens",
+    "accountHistory_filters_assetTypeTokenShowLess": "- Show Less Tokens",
+    "accountHistory_filters_assetTypeNFTShowAll": "+ Show All NFTs",
+    "accountHistory_filters_assetTypeNFTShowLess": "- Show Less NFTs",
+    "accountHistory_filters_assetTypeTokensLabel": "Tokens",
```

Comment:
I think we can ditch "assetType-" from these. And I'd say definitely remove the + and - from the string itself.